gammapy/data/observation_summary.py

```diff
@@ -0,0 +1,132 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from astropy.units import Quantity
+from astropy.coordinates import SkyCoord, Angle
+
+from .observation import ObservationTable
+
+__all__ = [
+    'ObservationTableSummary',
+]
+
+class ObservationTableSummary(object):
+    """Observation table summary.
+
+    Class allowing to summarize informations conatained in 
+    Observation index table (`~gammapy.data.ObservationTable`)
+
+    Parameters:
+    -----------
+    obs_table:  `~gammapy.data.ObservationTable`
+        Observation index table
+    target_pos: `~astropy.coordinates.SkyCoord`
+        Target position
+    """
+
+    def __init__(self, obs_table, target_pos=None):
+        self.obs_table = obs_table
+        self.target_pos = target_pos
+
+    @property
+    def offset(self):
+        """
+        Compute offsets of the different observations
+        
+        Returns
+        -------
+        vector : `~astropy.Angle`
+            Offsets
+        """
+        pnt_pos = SkyCoord(self.obs_table['RA_PNT'],
+                           self.obs_table['DEC_PNT'],
+                           unit='deg')
+
+        offset = pnt_pos.separation(self.target_pos)
+        
+        return offset.degree
```

Comment:
Calling `.degree` on an `Angle` object drops the units
```python
>>> from astropy.coordinates import Angle
>>> Angle(10, 'deg').degree
10.0
```
In this case I think you can just remove the `.degree` attribute access, `offset` is already an `Angle` object in degree.

For the future ... if you want to change units for an angle, but still have an `Angle` object, use the `.to` method:
```python
>>> Angle(1000, 'arcsec').to('deg')
<Angle 0.2777777777777778 deg>
```